src/components/PieChart/Legends.tsx

```diff
@@ -32,8 +32,9 @@ const List = styled.ul`
   display: flex;
   flex: 1;
   flex-direction: column;
-  cursor: pointer;
-  height: min-content;
+  margin: auto;
+  height: 100%;
+  overflow: scroll;
```

Comment:
```suggestion
  overflow-y: scroll;
```

maybe to avoid unwanted scrolls